From 38ca1a4cdcc13916e570a13f155940f946e29df1 Mon Sep 17 00:00:00 2001
From: Wright Feng <wright.feng@cypress.com>
Date: Fri, 18 Jan 2019 03:57:34 -0600
Subject: [PATCH 092/154] brcmfmac: remove the duplicate line of writing
 BRCMF_PCIE_REG_SBMBX

The code is duplicate so remove the line from pcie.c

Signed-off-by: Wright Feng <wright.feng@cypress.com>
---
 drivers/net/wireless/broadcom/brcm80211/brcmfmac/pcie.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/pcie.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/pcie.c
index e1454e0..63ad529 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/pcie.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/pcie.c
@@ -671,7 +671,6 @@ static int brcmf_pcie_exit_download_state(struct brcmf_pciedev_info *devinfo,
 
 	brcmf_pcie_write_tcm32(devinfo, addr, htod_mb_data);
 	pci_write_config_dword(devinfo->pdev, BRCMF_PCIE_REG_SBMBX, 1);
-	pci_write_config_dword(devinfo->pdev, BRCMF_PCIE_REG_SBMBX, 1);
 
 	return 0;
 }
-- 
1.9.1

